### PR TITLE
adding docker install instruction

### PR DESCRIPTION
**Pull request by kedif**
**When you open your pull requests, remove the lines that start and end with underscores (\_)**

## Purpose
how to install dockers.

## Approach
added the link to README.md

## Pre-Testing TODOs
links are not 404ing

## Testing Steps
make sure links go to right page

## Learning
installing docker
Closes #6

### CODE REVIEW

**Comment by fedluk70** [comment]
Nice job Kedi

**Review by fedluk70 — COMMENTED** [review]
Good job

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # DevOps-Apprenticeship-Notes
+k-6-youtube-workshops
+* Here is the docker post-install instructions that reduce the number of sudo commands: https://docs.docker.com/engine/install/linux-postinstall/
```
Comment (michaelachrisco) [inline README.md:2]: Nice job adding this link. That definitely helped us out when we were going through the initial install.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # DevOps-Apprenticeship-Notes
+k-6-youtube-workshops
```
Comment (michaelachrisco) [inline README.md]: Lets remove the branch names from the PR.

File: README.md [hunk 3]
```diff
@@ -11,6 +15,7 @@ https://www.youtube.com/watch?v=YcJ9IeukJL8:
 https://www.youtube.com/watch?v=SLB_c_ayRMo: 
 https://www.youtube.com/watch?v=gx311756mmmkdkdj
 
+development
```
Comment (michaelachrisco) [inline README.md]: Same here. Lets remove the branch names from the PR.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # DevOps-Apprenticeship-Notes
+k-6-youtube-workshops
+* Here is the docker post-install instructions that reduce the number of sudo commands: https://docs.docker.com/engine/install/linux-postinstall/
 
+*  adding some tfswitch vedio. Here is the tfswitch video link: https://drive.google.com/file/d/14xNJzpCajLEVzcGL5h3twfIyD-hRhkGc/view?usp=sharing
+=======
```
Comment (michaelachrisco) [inline README.md]: Im guessing you had a merge conflict. Please remove the `===` as they are only useful in the process of resolving the merge conflict.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 # DevOps-Apprenticeship-Notes
+k-6-youtube-workshops
+* Here is the docker post-install instructions that reduce the number of sudo commands: https://docs.docker.com/engine/install/linux-postinstall/
 
+*  adding some tfswitch vedio. Here is the tfswitch video link: https://drive.google.com/file/d/14xNJzpCajLEVzcGL5h3twfIyD-hRhkGc/view?usp=sharing
```
Comment (michaelachrisco) [inline README.md]: I would suggest fixing this spelling error and/or making this one sentence. 
```suggestion
* Here is the tfswitch video link: https://drive.google.com/file/d/14xNJzpCajLEVzcGL5h3twfIyD-hRhkGc/view?usp=sharing
```

When doing Pull Requests, feel free to take a look at your own submitted code/documentation and see if there is any other spelling/grammar issues.

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
@kedif Awesome job on the pull request! I like the inclusion of our `tfswitch` tool installation. 

I added a couple of recommended changes for now. Please take a look and see if they make sense. Thanks again for contributing!

File: README.md (hunk 4 above)
Comment (kedif) [reply README.md]: Thank you Micheal. I fixed everything that you asked. let me know if anything needs to change.

**Review by Ruhama21 — COMMENTED** [review]
Thank you!

**Review by fedluk70 — COMMENTED** [review]
Nice job Kedi

**Comment by michaelachrisco** [comment]
Awesome job, thanks for addressing the issues above! 

The only blocker I see now is the git conflict. Take a look here for a way to solve the conflict: https://www.freecodecamp.org/news/resolve-merge-conflicts-in-git-a-practical-guide/ 

Alternatively, you can have Github itself help with the git conflict via the "resolve conflicts" button above.

File: README.md [hunk 6]
```diff
@@ -1,14 +1,32 @@
...
+
+
 
 ## Contributing to this Repository
 
 # Take a look at our Bitwise Industries Standards and Practices for example markdown and best practices: https://github.com/Shift3/standards-and-practices
 
 
+
+k-6-youtube-workshops
```
Comment (michaelachrisco) [inline README.md:23]: Lets remove this branch reference

File: README.md [hunk 7]
```diff
@@ -1,14 +1,32 @@
...
+## Contributing to this Repository
 Terraform useful link: https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli
 
 Terraform video link: https://www.youtube.com/watch?v=YcJ9IeukJL8
 
+development
```
Comment (michaelachrisco) [inline README.md:29]: Same here. Lets remove this branch reference.

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
Your super close, looks good. I think we just need to remove the branch names and you should be good. Remove those branch names and then re-assign me to the PR.

**Comment by michaelachrisco** [comment]
Closing all old Prs over 1 month old. Feel free to re-submit!

**Pull request closed without merging**